src/GraphQL.Tests/Serialization/ExecutionErrorTests.cs

```diff
@@ -0,0 +1,69 @@
+namespace GraphQL.Tests.Serialization
+{
+    /// <summary>
+    /// Tests for <see cref="IGraphQLTextSerializer"/> implementations and the custom converters
+    /// that are used in the process of serializing an <see cref="ExecutionError"/> to JSON.
+    /// </summary>
+    public class ExecutionErrorTests
```

Comment:
I did not spend much time making a multitude of tests.  Any existing tests we have, within the entire codebase, that verifies serialized errors will use the new code, because it was moved (not copied) from `ExecutionResultJsonConverter` to `ExecutionErrorJsonConverter`.